cypress/e2e/alerts.cy.js

```diff
@@ -1,25 +1,52 @@
 describe('Cypress application', () => {
-  before(() => {
-
+  beforeEach(() => {
+    cy.visit('/alerts');
   });
 
   it('should have the ability to assert automatically resolved alerts', () => {
-
+    cy.get('#alertButton')
+      .click();
+    cy.on('window:alert', (text) => {
+      expect(text).to.equal('You clicked a button');
+    });
   });
 
-  it('should have the ability to assert scheduled allert', () => {
-
+  it('should have the ability to assert scheduled alert', () => {
+    cy.get('#timerAlertButton').click();
+    cy.wait(5000);
```

Comment:
Hi Olga. I like your tests. But please, bear in mind that we should avoid using cy.wait by all means xD Actually you can pass this test without using this command.

Reply:
Hello, Maxim! I think your comment is valid and I am pleased that you are objective.